src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx

```diff
@@ -0,0 +1,87 @@
+import { Image } from "@artsy/palette"
+import { DefaultHeaderArtwork_artwork } from "__generated__/DefaultHeaderArtwork_artwork.graphql"
+import { AnalyticsSchema } from "Artsy/Analytics"
+import { useTracking } from "Artsy/Analytics/useTracking"
+import React, { useCallback } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+const Link = styled.a`
+  display: flex;
+  align-items: flex-end;
+`
+
+export type DefaultHeaderArtworkProps = React.HTMLAttributes<
+  HTMLAnchorElement
+> & {
+  artwork: DefaultHeaderArtwork_artwork
+  small?: boolean
+  collectionId: string
+  collectionSlug: string
+}
+
+export const DefaultHeaderArtwork: React.FC<DefaultHeaderArtworkProps> = ({
+  artwork,
+  small = false,
+  collectionId,
+  collectionSlug,
+  ...rest
+}) => {
+  const { trackEvent } = useTracking()
+
+  const handleClick = useCallback(() => {
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.Click,
+      context_module: AnalyticsSchema.ContextModule.ArtworkBanner,
+      context_page_owner_type: AnalyticsSchema.OwnerType.Collection,
+      context_page: AnalyticsSchema.PageName.CollectionPage,
+      context_page_owner_id: collectionId,
+      context_page_owner_slug: collectionSlug,
+      destination_path: artwork.node.href,
+    })
+  }, [artwork, trackEvent])
+
+  return (
+    <Link
```

Comment:
This should use a [`<RouterLink to="...">`](https://github.com/artsy/reaction/blob/master/src/Artsy/Router/RouterLink.tsx) so that we can tap into the new app shell.